### DISCUSSION TITLE
object not persist

### DISCUSSION THREAD

**Discussion by agracia-foticos** [vuejs/pinia#2115] [Help and Questions] [answered]
### Reproduction

https://stackblitz.com/edit/nuxt-starter-b3gnpr

### Steps to reproduce the bug

In theory, the storage loads the object only once and the second time, since it's already loaded, it doesn't load it again.

However, in both cases the application variable is empty, when it should already be full.

### Expected behavior

"application empty we assign" should be painted only once

### Actual behavior

"application empty we assign" should be painted many times

**Comment by posva** [comment, marked as answer]
This is because Nuxt executes the action on the server and on the client

**Reply by agracia-foticos** [reply]
In Nuxt, the store Pinia data isn't load in Node / SSR  payload to hidrate in JS / Vue side?

**Reply by posva** [reply]
Since you are awaiting it, it does get hydrated but the action still gets called twice
